lib/manager/components/ProjectSettingsForm.js

```diff
@@ -356,53 +358,61 @@ export default class ProjectSettingsForm extends Component<Props, State> {
               </ControlLabel>
               <TimezoneSelect
                 value={model.defaultTimeZone}
-                onChange={this._onChangeTimeZone} />
+                onChange={this._onChangeTimeZone}
+              />
+            </ListGroupItem>
+          </ListGroup>
+        </Panel>
+        <Panel header={<h4>Local Places Index</h4>}>
+          <ListGroup fill>
+            <ListGroupItem>
+              <FormGroup validationState={validationState(validation.webhookUrl)}>
+                <ControlLabel>Webhook URL</ControlLabel>
+                <FormControl
+                  value={model.peliasWebhookUrl || ''}
+                  name={'peliasWebhookUrl'}
+                  onChange={this._onChangeTextInput}
+                />
+                <FormControl.Feedback />
+              </FormGroup>
             </ListGroupItem>
           </ListGroup>
         </Panel>
         {showDangerZone &&
-          <Panel bsStyle='danger' header={<h3>Danger zone</h3>}>
-            <ListGroup fill>
-              <ListGroupItem>
-                <Button
-                  bsStyle='danger'
-                  className='pull-right'
-                  data-test-id='delete-project-button'
-                  onClick={this._onDeleteProject}
-                >
-                  <Icon type='trash' /> Delete project
-                </Button>
-                <h4>Delete this project.</h4>
-                <p>Once you delete an project, the project and all feed sources it contains cannot be recovered.</p>
-              </ListGroupItem>
-            </ListGroup>
-          </Panel>
-        }
+        <Panel bsStyle='danger' header={<h3>Danger zone</h3>}>
+          <ListGroup fill>
+            <ListGroupItem>
+              <Button
+                bsStyle='danger'
+                className='pull-right'
+                data-test-id='delete-project-button'
+                onClick={this._onDeleteProject}>
+                <Icon type='trash' /> Delete project
+              </Button>
+              <h4>Delete this project.</h4>
+              <p>Once you delete an project, the project and all feed sources it contains cannot be recovered.</p>
+            </ListGroupItem>
+          </ListGroup>
+        </Panel>}
```

Comment:
Revert the indenting change.

Reply:
Yeah prettier did some strange things here... and that was using format selection as well! it does even stranger things when run on the entire file. Un-indented in d54487d0